controllers/openstackprovisionserver_controller.go

```diff
@@ -266,6 +244,13 @@ func (r *OpenStackProvisionServerReconciler) Reconcile(ctx context.Context, req
 				cond.Type = shared.ProvisionServerCondTypeProvisioned
 				cond.Reason = shared.OpenStackProvisionServerCondReasonProvisioned
 				cond.Message = "OpenStackProvisionServer has been provisioned"
+
+				// Only set the localImageURL if the pod has finished init
```

Comment:
Working now, localImageURL gets cleared while provisioning (and qemu-img convert is running) and set once the pod is fully initialized